modules/social_features/social_event/modules/social_event_managers/social_event_managers.module

```diff
@@ -65,6 +65,40 @@ function social_event_managers_activity_send_email_notifications_alter(array &$i
   }
 }
 
+/**
+ * Implements hook_entity_access().
+ *
+ * This allows view access to event_enrollment entities for users which are the
```

Comment:
This comment is true for now, but might change in the future, how about moving it inside the function so it will be more visible if something changes?